internal/build/phases.go

```diff
@@ -29,6 +29,66 @@ type PhaseFactory interface {
 	New(provider *PhaseConfigProvider) RunnerCleaner
 }
 
+func (l *Lifecycle) Create(ctx context.Context, publish, clearCache bool, runImage, launchCacheName, cacheName, repoName, networkMode string, phaseFactory PhaseFactory) error {
+	var configProvider *PhaseConfigProvider
+	var binds []string
+
+	args := []string{
```

Comment:
You're right. We're not passing the flags you mentioned since we're using their default values.
I don't mind changing the other phases but I'm not sure it should be part of this PR.
What do you think?

Reply:
I can see how this would be its own unit of work. If this is something that we want to do, we should probably create an issue for it so that it doesn't get lost.

Reply:
I wrote it to myself. Thanks!

Reply:
Please see: https://github.com/buildpacks/pack/issues/610